README.md

```diff
@@ -89,6 +89,18 @@ Inquire if the type of local authentication policy is supported on this device w
 
 Get what type of hardware biometry support the device has. Resolves to a `Keychain.BIOMETRY_TYPE` value when supported, otherwise `null`.
 
+### `getSecurityLevel()`
```

Comment:
please mention also here that this is android-only

Reply:
done